src/lib/search-params.ts

```diff
@@ -125,6 +132,7 @@ export function fromSearchParamsToSearchOptions({
     filters: {
       publishers: publishers?.split(',').filter(id => !!id),
       licenses: licenses?.split(',').filter(id => !!id),
+      spatialCoverages: spatialCoverages?.split(',').filter(id => !!id),
```

Comment:
Rule of three? 😄

Reply:
Yes! I did not do this here yet, because I started a bigger refactor of this function using `zod.transform`; see the unfinished branch: https://github.com/colonial-heritage/dataset-browser/blob/062565ffbc7db05f5f2bf87c3c4529f87c73a071/src/lib/search-params.ts#L32

Is it ok to wait for the refactor?

Reply:
Sure, no problem!